server/routes/index.js

```diff
@@ -1290,4 +1292,73 @@ router.route('/charts/new')
       return res.status(500).send({ message: err.message })
     }
 })
+
+/**
+ * Study Details
+ */
+router.route('/study-details')
+  .get(ensureAuthenticated, async (req, res) => {
+    try {
+      const { display_name, role, icon } = req.session;
+      return res.status(200).send(studyDetailsPage({
+        uid: req.user,
```

Comment:
The req.user value is set by passport via the serialization and deserialization of the user. It could be an entire object that's stored, but in this application they decided to go with the uid. 

https://github.com/AMP-SCZ/dpdash/blob/19e87a7067e6d9ab1b385d9f8a27b9bd00f5f858/server/utils/passport/ldap.js#L102-L108

The rest of the user object values are stored in the session. 

https://github.com/AMP-SCZ/dpdash/blob/19e87a7067e6d9ab1b385d9f8a27b9bd00f5f858/server/utils/passport/ldap.js#L197-L201